src/wp-includes/blocks/cover.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
I have vague memories suggestion that this one was a plugin only thing while waiting for a core change. but it's old, we may have to check the original PR.

The fact that it's an old thing in the plugin that is not in Core suggests that we could avoid it?

Reply:
I see that @mtias authored https://github.com/WordPress/gutenberg/pull/22346 that added this change.

It's already in WP 5.6:

https://github.com/WordPress/wordpress-develop/blob/9ec7be02d1d5985e35caba72de784a651a0dae9d/src/wp-includes/kses.php#L406

So we should omit it here and completely remove from the Gutenberg plugin and WordPress packages.

Reply:
Makes sense 👍

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/1226/commits/d8becf7e4e99abb80fb6fcbb9298f51a180134a0.

Reply:
PR in Gutenberg: https://github.com/WordPress/gutenberg/pull/31503.